neuralprophet/forecaster.py

```diff
@@ -749,6 +749,7 @@ def add_events(
     def add_country_holidays(
         self,
         country_name: Union[str, list],
+        subdivision_name: Optional[Union[str, dict]] = None,
```

Comment:
seems like the subdivision commits slipped in here. 
we can either remove those commits or merge that PR first and be careful that these lines do not longer reappear due to commit squashing.